### ISSUE TITLE
Add showToastSuccess() Pop-downs to several areas of the interface

### ISSUE THREAD

**Issue by DaleMcGrew** [wevote/WebApp#1304]
### Please describe the issue (What happens? What do you expect?)
In WebApp/src/js/components/Widgets/ItemActionBar.jsx we use showToastSuccess() to pop down a message when you do an action.

Please import this package:
import { showToastError, showToastSuccess } from "../../utils/showToast";

...and cause messages to appear in all of these cases:
- [ ] On the Ballot page, when you click and unclick the bookmark
- [ ] When you "Listen" or stop "Listening" to any organization or person
- [ ] When you "Ignore" any organization or person
- [ ] When you make your opinion public or private:
![screen shot 2018-02-01 at 9 00 20 pm](https://user-images.githubusercontent.com/7756031/35717273-ff18627a-0792-11e8-96d4-f92862b4001a.png)
- [ ] When you support or oppose a ballot item on the candidate or office page (there is currently a "Event is not registered." error in the browser console when you support or oppose from the candidate or office page)
- [ ] When you Follow or stop Following an Issue from the Network page

**Comment by uncleoptimus** [comment]
Just to be clear, when we un-click to remove some input state we want to show the red "error" toast?

**Comment by DaleMcGrew** [comment]
Yes, that sounds good. With one exception: lets keep Support and Stop Supporting both green, and Oppose and Stop Opposing both red.

**Comment by uncleoptimus** [comment]
Bookmark component updated 👍 
Messaging currently "Bookmark set!" and "Bookmark removed!", respectively.

Opinions/Positions component updated.
Messaging currently shows
- "Positions now visible to friends only!"
- "Your positions are now public!"

Questions:

On *Listen/Stop listening*, this is on the Network page?
Ditto *Ignore/Stop ignoring*?

Note:
Moved the `ToastContainer` to a higher level in the app so that various views can show toasts. I noticed from `Application.js` that there are three types of App views: 'theater', 'voter-guide' and default. I only added the `ToastContainer` to default view since it looks like the other view types wouldn't have toast-interactions available?

**Comment by uncleoptimus** [comment]
Added toasts to the `FollowToggle` component.

Messaging currently:
- "Now following!"
- "No longer following!"

`FollowToggle` seems to be used in multiple views so I am unsure if the messaging fits all situations 🤔 

---
Also added toast to "ignore" button.

Message: "Added to ignore list."

**Comment by DaleMcGrew** [comment]
Hi @uncleoptimus, feel free to make partial commits with your changes to date for this issue. Thank you!

**Comment by uncleoptimus** [comment]
@DaleMcGrew can I get clarification on the 5th bullet item:
>When you support or oppose a ballot item on the candidate or office page (there is currently a "Event is not registered." error in the browser console when you support or oppose from the candidate or office page)

This is not the thumbs-up/down "support/oppose" buttons that already spawn toasts on click?
I am looking at the Ballot view with various offices and propositions listed.

**Comment by DaleMcGrew** [comment]
Hi @uncleoptimus, this is on pages like these two:
https://wevote.us/candidate/wv02cand23994/b/btdb/
https://wevote.us/office/wv02off11651/b/btdb/
The toast notifications are not currently appearing, and instead we get the error "<SHOW_TOAST> Event is not registered. Did you forgot to bind the event ?" You may have fixed this on your local build when you "Moved the ToastContainer to a higher level in the app so that various views can show toasts."

**Comment by uncleoptimus** [comment]
Gotcha 👍  

Yeah I saw the issue on the production page. The toasts actually did pop up but only after navigating back to the Ballot page.

This should indeed be addressed now that ToastContainer is lifted ☕️
